src/main/java/view/View.java

```diff
@@ -0,0 +1,39 @@
+package view;
+
+import util.CompareAnswer;
+
+import java.util.Scanner;
+
+import static util.Constant.*;
+
+/**
+ * 화면출력용 클래스
+ */
+public class View {
+    public static void printStart() {
+        System.out.println(GAME_START_MESSAGE);
+    }
+
+    public void printResult(CompareAnswer answer) {
+        if (answer.getStrikeCount()>STRIKE_BALL_MIN) {
```

Comment:
인텔리제이에서 자동 줄 정렬 기능을 제공하고 있습니다! 

https://penthegom.tistory.com/47

```suggestion
        if (answer.getStrikeCount() > STRIKE_BALL_MIN) {
```